Cargo.toml

```diff
@@ -74,6 +74,10 @@ test = true
 #gfx-backend-dx11 = { version = "0.5.0", path = "../gfx/src/backend/dx11" }
 #gfx-descriptor = { version = "0.1.0", path = "../gfx-extras/gfx-descriptor" }
 #gfx-memory = { version = "0.1.0", path = "../gfx-extras/gfx-memory" }
+wasm-bindgen = { git = "https://github.com/rustwasm/wasm-bindgen" }
```

Comment:
shouldn't we pick a concrete revision? Otherwise, I'm not sure if Cargo is even obligated to grab the latest git, it can still use some older revisions for some users.

Reply:
Yeah definitely, we should include `rev`